### ISSUE TITLE
Paragraph line height in default theme does not meet accessibility guidelines

### ISSUE THREAD

**Issue by Blackbaud-ToddRoberts** [blackbaud/skyux-theme#175]
### Expected behavior
paragraphs have line-height: 1.5

### Actual behavior
paragraphs have no specified line height.

Talking to @Blackbaud-PaulCrowder the safest approach would be to add a new class for "sky-paragraph" to avoid blowing up existing implementations.

[AB#1551357](https://blackbaud.visualstudio.com/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/1551357)
